### ISSUE TITLE
Spine containing same spine item multiple times does not generate error/alert

### ISSUE THREAD

**Issue by rdeltour** [w3c/epubcheck#182]
_From [elmimmo@gmail.com](https://code.google.com/u/elmimmo@gmail.com/) on May 24, 2012 11:07:14_

PROBLEM: When validating against EPUB version 2.0, epubcheck will not alert that a specific spine item appears more than once in spine, which OPF 2.0.1 explicitly prohibits:

> In addition, a specific spine item (from the perspective of its id attribute value in manifest) must not appear more than once in spine.

Not that I agree one bit with this rule (what if a same resource should be indeed shown several times throughout the books' reading order; this is bonding e-books to pointless paper-based rules), but at any rate it is that is what the spec says. METHOD: 1. Create an EPUB 2, the OPF document of which contains a spine which lists the item twice, like:

```
  <itemref idref="chapter1"/>
  <itemref idref="chapter2"/>
  <itemref idref="chapter1"/>
```
1. Run epubcheck on it. RESULT: epubcheck will report:

> No errors or warnings detected EXPECTED: I expected epubcheck to report:
> 
> Spine item "chapter1" appears more than once in spine. PLATFORM: epubcheck 3.0b5, Mac OS 10.7.4

_Original issue: http://code.google.com/p/epubcheck/issues/detail?id=182_
